BAZProject/MainMovie/MainMovieViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MainMovieViewController.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MainMovieViewController: UITabBarController, MainMovieViewProtocol {
+
+	var presenter: MainMoviePresenterProtocol?
+
+    private var mainMovieTabs: [MainMovieTab] = []
```

Comment:
Ha que arreglar la indentación aquí.

```suggestion
    var presenter: MainMoviePresenterProtocol?

    private var mainMovieTabs: [MainMovieTab] = []
```